hotrod/src/main/java/io/hyperfoil/hotrod/steps/HotRodCacheNameBuilder.java

```diff
@@ -0,0 +1,22 @@
+package io.hyperfoil.hotrod.steps;
+
+import io.hyperfoil.api.config.BuilderBase;
+import io.hyperfoil.api.session.Session;
+import io.hyperfoil.function.SerializableFunction;
+
+@FunctionalInterface
+public interface HotRodCacheNameBuilder extends BuilderBase<HotRodCacheNameBuilder> {
```

Comment:
You do not need to create a new builder to provide strings; use `StringGeneratorImplBuilder` (okay I admit the name is somewhat clunky).

Reply:
Fixed with: Key and Value Generator